lightbluetent/society.py

```diff
@@ -26,6 +28,109 @@ def welcome(uid):
     if society.logo == current_app.config["DEFAULT_LOGO"]:
         has_logo = False
 
+    meeting = AttendeeMeeting(society.bbb_id, society.attendee_pw)
+    running = meeting.is_running()
+
+    if request.method == "POST":
+
+        # We should only be able to submit a POST request if the meeting is running.
+        # TODO: there may be circumstances where the meeting was running when this page was
+        # loaded, but it is no longer running when we submit this POST request. How should
+        # we handle this, if at all? Currently we abort(500).
```

Comment:
I believe we should check internally if the meeting is still running by calling the is_running() utility.

Reply:
We do, the issue is what to do when is_running() returns False.

Reply:
Ah, I see. I suggest we redirect to the error page and pass in a custom error object with the needed params, eg error.code = 404, error.name = "Meeting not found"

Reply:
9fbaa7a now flashes a message that says the meeting is no longer running. Does this work?